src/components/Icon/Icon.stories.tsx

```
@@ -0,0 +1,68 @@
+};
+
+export const Tertiary: Story = {
+  args: {
+    iconType: 'inputDelete',
+    iconColor: 'tertiary',
+  },
+};
+
+export const Error: Story = {
+  args: {
+    iconType: 'inputDelete',
```

Biome (1.9.4)

[error] Do not shadow the global "Error" property.

Consider renaming this variable. It's easy to confuse the origin of variables when they're named after a known global.

(lint/suspicious/noShadowRestrictedNames)